util/trelloinit.py

```diff
@@ -0,0 +1,74 @@
+""" Trello config initializer.

```

Comment:
This is a nice little script, but I'm not convinced it's nicer than simply asking the users to specify the relevant variables in their environment file! Sticking with the `site_trello.json` file approach means:
* Our board/list ids end up baked inside our Docker images - not critical but feels unnecessary
* If we want to change those, we need to do so before we build an image, rather than sharing the image between environments and just varying the env variables

Reply:
yeah.  The thing keeping this alive is that the app takes a dictionary, which I can't express in environment variables.  I'm not saying it's a _good_ reason, only that's why it persists.